internal/controllers/clustertemplate_controller.go

```diff
@@ -339,3 +416,24 @@ func (r *ClusterTemplateReconciler) enqueueClusterTemplatesForConfigmap(ctx cont
 	}
 	return requests
 }
+
```

Comment:
If you decide to keep these functions, I think they should be moved to the utils package.

Reply:
Fixed